NEWS.md

```diff
@@ -305,7 +305,7 @@ changelog for details of the default gems or bundled gems.
     * added: `rb_postponed_job_preregister()`
     * added: `rb_postponed_job_trigger()`
     * deprecated: `rb_postponed_job_register()` (and semantic change. see below)
-    * deprecated: `rb_postponed_job_register_once()`
```

Comment:
Eurgh, sorry about this and thank you for fixing it

Reply:
Not at all! The `preregister` improvement looks great.